vscode-trace-webviews/src/style/trace-explorer.css

```diff
@@ -0,0 +1,12 @@
+:root {
```

Comment:
Should it be trace-viewer.css?

Reply:
I used the corresponding name as in theia-trace-extension where also some clean-up needs to be done. Makes sense to change it here. Please note that the styling in this patch works for now. But some updates are required in the theia-trace-extension's react-components to support better styling and theming independent of the "theia" reference in the variables.